src/partisan/irods.py

```diff
@@ -841,14 +849,33 @@ def __init__(self, attribute: Any, value: Any, units=None, namespace=None):
         if namespace is None:
             namespace = ""
 
-        if namespace.find(AVU.SEPARATOR) >= 0:
+        attr = str(attribute)
+        value = str(value)
+
+        if re.match(r"\s+$", attr):
+            raise ValueError("AVU attribute may not be entirely whitespace")
```

Comment:
I couldn't find whether they are already covered, do you also need to add tests for this type of strings with whitespaces?